StatusEffect/EverLeaking

```diff
@@ -0,0 +1,57 @@
+extends StatusEffectBase
+
+var _toCheck = [
+	character.hasBodypart(BodypartSlot.Penis),
+	character.hasBodypart(BodypartSlot.Vagina),
+	character.hasWombIn(BodypartSlot.Anus)
```

Comment:
I'm surprised that this works (if it does)
also you forgot to add .gd at the end of the file